docs/content/guides/operator/archives.mdx

```diff
@@ -3,107 +3,34 @@ title: Sui Archives
+A Sui archive is a history of all transaction data on Sui. As a Sui node operator, you can write this history to an object store like S3, GCS, or similar for safe keeping. Saving this archive data is considered a best practice because Sui prunes transactions on Full nodes to remove historical transactions and their effects. Peer nodes, for example, might not catch up with all transactions and effects through synchronization if they lag behind the current epoch by more than the latest few epochs. Instead of relying on synchronization, peer nodes can fallback to downloading this information from an archive.
 
-Sui Archival nodes (Full nodes that write to an archive) don't store historical state on local storage and don't help query historical data. They serve the purpose of enabling peer nodes to catch up to the latest checkpoint and are useful for auditing and verifying the complete history of all transactions on the network.
-
-## Create an Archival node
-
-To start storing transaction history as an archive, you need to modify your node configuration. Open your fullnode.yaml file and add the following configuration. By default, the fullnode.yaml file is located in your `~/.sui/sui_config` directory.
-
-```yaml
-state-archive-write-config:
-  object-store-config:
-    object-store: "S3"
-    bucket: "<bucket_name>"
-    aws-access-key-id: "<AWS_ACCESS_KEY_ID>"
-    aws-secret-access-key: "<AWS_SECRET_ACCESS_KEY>"
-    aws-region: "<aws_region>"
-    object-store-connection-limit: 20
-  concurrency: 5
-  use-for-pruning-watermark: false
-state-archive-read-config:
-  - object-store-config:
-      object-store: "S3"
-      # Use the same bucket which is being used in `state-archive-write-config`
-      bucket: "<bucket_name>"
-      aws-access-key-id: "<AWS_ACCESS_KEY_ID>"
-      aws-secret-access-key: "<AWS_SECRET_ACCESS_KEY>"
-      aws-region: "<aws_region>"
-      object-store-connection-limit: 20
-    concurrency: 5
-    # This should be set to true in this case. Setting this to true
-    # would prevent pruning of local transaction data until it is archived
-    # in the bucket
-    use-for-pruning-watermark: true
-```
+:::info
+Starting with the 1.49 release, the existing special archive format is being deprecated in favor of a generic data ingestion format. Please migrate your configs using the configuration example listed below
+:::
 
 ## Set up archival fallback {#set-up-archival-fallback}
```

Comment:
Again, I think we should move the current documentation for fallback to a section like `Legacy archival process (deprecated)` or something similar. And then after a couple of releases, we could totally remove it.

The new section referring to the checkpoint buckets could be named `Set up archival fallback (latest)` or similar, and then once we remove the legacy portion after a couple of releases, we could remove the `(latest)` from here.